packages/serp-report/src/content_scripts/serp-report.js

```diff
@@ -54,76 +54,35 @@ function renderPopup(container, stats) {
   document.body.appendChild(wrapper);
 }
 
-function renderWheel(WTMTrackerWheel, anchor, stats) {
+function getWheelElement(WTMTrackerWheel, stats) {
   const count = stats.stats.length;
 
   if (count === 0) {
-    return;
+    return null;
   }
 
-  const parent = anchor.parentElement;
-  parent.style.position = 'relative';
-  const threeDotsElement = parent.querySelector(
-    'div[jsslot] div[aria-haspopup], div[jsaction] div[role=button] span',
-  );
-
-  if (!threeDotsElement && isMobile) {
-    // that's not a "full" result
-    return;
-  }
   const container = document.createElement('div');
   container.classList.add('wtm-tracker-wheel-container');
-  if (isMobile) {
-    container.style.left =
-      threeDotsElement.getBoundingClientRect().left -
-      parent.getBoundingClientRect().left -
-      50 +
-      'px';
-    container.style.top = 15 + 'px';
-  } else if (threeDotsElement) {
-    // default path on Safari (Desktop)
-    container.style.left =
-      threeDotsElement.getBoundingClientRect().right -
-      parent.getBoundingClientRect().left +
-      5 +
-      'px';
-  } else {
-    // default path in Chrome
```

Comment:
Won't we break the functionality for Desktop in the ghostery-extension if we remove the logic for Chrome (Desktop)?

Reply:
I've used the more appropriate place to put the wheel, so hacks with finding structure like above are no longer required.